src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -733,6 +782,8 @@ InputsMap getInputMap() {
 #include "ngraph/opsets/opset8_tbl.hpp"
 #include "ngraph/opsets/opset9_tbl.hpp"
 
+NGRAPH_OP(AUGRUCell, ov::op::internal)
```

Comment:
Are defined they in one header file (as `opset9_tbl` and etc)? 
These ops are not part of public opset. I am not sure that it's correct place to check it here. Moreover this code looks as a workaround.

Reply:
@tomdol @ilyachur @jane-intel @mbencer Does anyone have any idea how the private ops should be included? There is no header file that has them.

Reply:
Can we include this internal operation via include directive? So no need to create new opset_tbl header file for private operations.

Reply:
What is the "include directive"? Can you have more details please @rkazants ?

Reply:
updated with opset_private_tbl.hpp.